moreComfortable.js

```diff
@@ -0,0 +1,114 @@
+//two players will take turns to play
+//when the player clicks submit, game rolls 2 dice and show the values
+//player then can pick the order of dice they want ie: 36 or 63
+//after both players have rolled and chosen the dice order, the player with the highest combined number wins.
+//have to store the values of the chosen combination
+var player1Choice = [0];
+var player2Choice = [0];
+
+//store dice values
+var diceValues = [];
+//since there will be two players, multiple program states needed
+var currentGameMode = "waiting for player 1 input";
+//rolling dice function
+var numberGenerator = function () {
+  var randomDecimal = Math.random() * 6;
+  var randomInteger = Math.floor(randomDecimal);
+  var randomNumber = randomInteger + 1;
+  return randomNumber;
+};
+//we can use loops to store the dice roll values into the array
+var dicePlayer1 = function () {
+  var counter = 0;
+  while (counter < 2) {
+    var rolledValue = numberGenerator();
+    diceValues.push(rolledValue);
+    counter += 1;
+  }
+  return `Hi Player 1, you have rolled ${diceValues[0]} and ${diceValues[1]}. <br><br> Please choose the order you would like to combine your values. <br> <br> (1) for normal and (2) for reversed. `;
+};
+//we can use loops to store the dice roll values into the array
+var dicePlayer2 = function () {
+  var counter = 0;
+  while (counter < 2) {
+    rolledValue = numberGenerator();
+    diceValues.push(rolledValue);
+    counter += 1;
+  }
+  return `Hi Player 2, you have rolled ${diceValues[2]} and ${diceValues[3]}. <br> <br> Please choose the order you would like to combine your values. <br> <br> (1) for normal and (2) for reversed. `;
+};
+//MAIN FUNCTION
+var main = function (input) {
+  var myOutputValue = "";
+  if (currentGameMode == "waiting for player 1 input") {
+    myOutputValue = dicePlayer1();
+    currentGameMode = "waiting for player 1 to choose dice order";
+
+    return myOutputValue;
+  } else if (currentGameMode == "waiting for player 2 input") {
+    myOutputValue = dicePlayer2();
+    currentGameMode = "waiting for player 2 to choose dice order";
+
+    return myOutputValue;
+  }
+  if (currentGameMode == "waiting for player 1 to choose dice order") {
+    //have to concatenate the two values together
+    if (input == "1") {
+      currentGameMode = "waiting for player 2 input";
+      var player1 = Number(String(diceValues[0]) + String(diceValues[1]));
+      player1Choice[0] = player1Choice[0] + player1;
```

Comment:
You do not have to assign var player1Choice = [0] at line 1 and then add player1's choice.
A better way would be to declare player1Choice as an empty array at line 1 and then use the .push method to add player 1's score to the array over here. 

.push(): https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push